hs_core/page_processors.py

```diff
@@ -120,6 +125,11 @@ def get_page_context(page, user, resource_edit=False, extended_metadata_layout=N
     has_web_ref = res_has_web_reference(content_model)
 
     keywords = json.dumps([sub.value for sub in content_model.metadata.subjects.all()])
+    if settings.COMMUNITIES_ENABLED:
```

Comment:
nope, this loads topics from the database for all resources.  We shouldn't be doing something globally that's only intended for a small set of users.

Reply:
Oversight. The conditional should be checking COMMUNITIES_ENABLED as well as something like "member of CZO community"

Reply:
I was referring to the global call of collecting all topics.  Sounds like topics may or may not be a CZO only thing right now.  If it is decided to be CZO only, then I guess we can leave this as is.  If not, then it's going to have to be collected by community.

Reply:
Won't know until you have the third Community, so probably should stay as-is for now, as technically this will be used by all Communities. This is the Agile process of introducing a new concept to a product. The beta group will help refine functionality and as more Communities come online, details like this can be iterated on. I think we agree, we've moved on from Waterfall development of establishing full functionality for all cases and implementing upfront. Things take too long that way.

Reply:
I don't think my comment excludes agile development.  I'm marking as resolved because it sounds like we want topics to be global for now.